tracee/tracee.go

```diff
@@ -922,7 +923,13 @@ func readArgFromBuff(dataBuff io.Reader) (interface{}, error) {
 		if err != nil {
 			return nil, err
 		}
-		res = fmt.Sprintf("%v", sockaddr)[3:] // remove the leading "map" string
+		var s string
+		for key, val := range sockaddr {
+			s += fmt.Sprintf("'%s': '%s',", key, val)
```

Comment:
i think it's better to use `"` instead of `'` for strings (as it's json compliant and more common in different parsers)